doc/source/user_guide/style.ipynb

```diff
@@ -61,21 +50,23 @@
    "outputs": [],
    "source": [
     "import pandas as pd\n",
+    "from pandas.io.formats.style import Styler\n",
```

Comment:
can you move this to the subclassing section / where its actually used (as its not normally needed for most operations)